docs/add-ins/get-and-set-recurrence.md

```diff
@@ -0,0 +1,190 @@
+
+Along with the recurrence pattern, you also need to determine the start and end dates/times of your appointment series. The [`SeriesTime`][SeriesTime link] object is used to manage that information.
+
+The appointment organizer can set up the appointment series in compose mode only. In the following example, the appointment series is set to occur from 10:30 AM to 11:00 AM PST every Tuesday and Thursday during the period November 2, 2017 to December 2, 2017.
+
+```js
+var seriesTimeObject = new Office.SeriesTime();
+seriesTimeObject.setStartDate(2017,10,2);
+seriesTimeObject.setEndDate(2017,11,2);
+seriesTimeObject.setStartTime(10,30);
+seriesTimeObject.setDuration(30);
+
+var pattern = {
+    "seriesTime": seriesTimeObject,
+    "recurrenceType": "weekly",
+    "recurrenceProperties": {"interval": 1, "days": ["tue", "thu"], "firstDayOfWeek": "sun"},
+    "recurrenceTimeZone": {"name": "Pacific Standard Time"}};
+
+Office.context.mailbox.item.recurrence.setAsync(pattern, callback);
+
+function callback(asyncResult)
+{
+    console.log(JSON.stringify(asyncResult));
+}
+```
+
+> [!NOTE]
+> Only the appointment organizer can set the recurrence.
+
+## Get recurrence
+
+### Get recurrence as the organizer
+
+In the following example in compose mode, the appointment organizer can get the recurrence object of an appointment series given the series item or an instance of that series.
+
+```js
+Office.context.mailbox.item.recurrence.getAsync(callback);
+
+function callback(asyncResult){
+    var context = asyncResult.context;
+    var recurrence = asyncResult.value;
+
+    if (recurrence == null) {
+        console.log("Non-recurring meeting");
+        return;
+    }
+
+    console.log(JSON.stringify(recurrence));
+}
+```
+
+The following is an example of the results of the `getAsync` call on a series.
+
+> [!NOTE]
+> In this example, the `seriesTimeObject` is a placeholder for the JSON representing the `recurrence.seriesTime` property. You should use the [`SeriesTime`][SeriesTime link] methods to get the recurrence date and time properties.
+
+```js
+Recurrence = {
+    "recurrenceType": "weekly",
+    "recurrenceProperties": {
+        "interval": 1,
+        "days": ["tue","thu"],
+        "firstDayOfWeek": "sun"},
+    "seriesTime": {seriesTimeObject},
+    "recurrenceTimeZone": {
+        "name": "Pacific Standard Time",
+        "offset": -480}}
+```
+
+### Get recurrence as an attendee
+
+In the following example, an appointment attendee can get the recurrence object of an appointment series given the series, an instance of that series, or a meeting request.
+
+```js
+outputRecurrence(Office.context.mailbox.item);
+
+function outputRecurrence(item) {
+    var recurrence = item.recurrence;
+    var seriesId = item.seriesId;
+
+    if (recurrence == null) {
+        console.log("Non-recurring item");
+        return;
+    }
+
+    console.log(JSON.stringify(recurrence));
+}
+```
+
+The following is an example of the `item.recurrence` property.
```

Comment:
Suggest rephrasing this sentence to be more active-voice...perhaps something like -> "The following example shows the value of the `item.recurrence` property for an appointment series."